blockchain/blockchain.go

```diff
@@ -1424,3 +1425,11 @@ func calculateLogsBloom(cfg config.Config, height uint64, receipts []*action.Rec
 	}
 	return bloom
 }
+
+func calculateTxNumber(cfg config.Config, height uint64) uint64 {
+	if height < cfg.Genesis.BeringBlockHeight {
+		return 0
+	}
+	tn := height
```

Comment:
You want to provide num of actions instead?

Reply:
Committed as new version.